src/components/books.jsx

```diff
@@ -0,0 +1,50 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+import Book from './bookItem';
+
+const Books = () => {
+  const library = useSelector((store) => store.library);
+  const dispatch = useDispatch();
+  const submitBook = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: v4(),
+      title: e.target.bookTitle.value,
+      author: 'Leonardo Pareja',
+    };
+    dispatch(addBook(newBook));
+    e.target.bookTitle.value = '';
+    e.target.categories.value = '';
+  };
+
+  // const deleteBook = (id) => {
+  //   useDispatch(removeBook(id));
+  // };
```

Comment:
Let's remove this commented code in order to make your code look cleaner ✊

Reply:
Thanks for the suggestion.